sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -8,14 +8,14 @@
     <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
     <Nullable>enable</Nullable>
     <DefineConstants>$(DefineConstants);AZURE_NULLABLE;HAS_INTERNALS_VISIBLE_CORE</DefineConstants>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net461;net472;netcoreapp2.1;net5.0;net6.0</TargetFrameworks>
+    
+    <!-- TODO: is it really ok to drop support for netcoreapp2.1? -->
```

Comment:
Per @petermarcu, we believe this is ok, as long as we document that we dropped support for older runtimes in some version and probably bump everything that does it by a minor version at the same time.

Reply:
Force case I think anyone that was depending on it would fallback to the netstandard2.0 version of the library.

Reply:
@KrzysztofCwalina FYI